server/handle_policy_eval.go

```diff
@@ -90,30 +92,23 @@ func policyEvaluateHandler(_ *config.Conf) gin.HandlerFunc {
 			return
 		}
 
-		decision, err := policyQuery(cmd.Rego.Policy, cmd.Rego.IsAllow, cmd.Input)
+		decision, err := policyQuery(cmd.Policy, cmd.Input)
 		if err != nil {
 			abortWithError(c, 500, err.Error())
 			return
 		}
 
 		c.JSON(200, gin.H{
-			"isAllow":      cmd.Rego.IsAllow,
 			"isSuccessful": decision,
 		})
 		return
 	}
 }
 
-func policyQuery(policyRego string, isAllow bool, input interface{}) (decision bool, err error) {
-
-	policyRegoEx := fmt.Sprintf("package example.auth\n\n%v", policyRego)
-	var policyQuery string
-	if isAllow {
-		policyQuery = fmt.Sprintf("data.example.auth.allow")
-	} else {
-		policyQuery = fmt.Sprintf("data.example.auth.deny")
-	}
+func policyQuery(policyRego string, input interface{}) (decision bool, err error) {
 
+	policyRegoEx := fmt.Sprintf("package policyman.auth\n\n%v", policyRego)
```

Comment:
do we need to append this? What I thought is, since we are loading the string fully, the rego must contain the package name already.

Reply:
If we let the users define the package name by themselves, we will not be able to be aware of which package to evaluate, or we must  extract the package name from the rego files,

Reply:
@iskey should we only then control if the package name already exists and overwrite it?